testsuite/ada_lsp/SA24-055.called_by_on_abstract/test.json

```diff
@@ -14,45 +14,113 @@
+                     "documentHighlight": {},
+                     "documentSymbol": {
+                        "hierarchicalDocumentSymbolSupport": true
+                     },
+                     "codeLens": {},
+                     "colorProvider": {},
+                     "formatting": {
+                        "dynamicRegistration": false
+                     },
+                     "rangeFormatting": {
+                        "dynamicRegistration": false
+                     },
+                     "onTypeFormatting": {
+                        "dynamicRegistration": false
+                     },
+                     "foldingRange": {
+                        "lineFoldingOnly": true
+                     },
+                     "selectionRange": {},
+                     "linkedEditingRange": {},
+                     "callHierarchy": {},
+                     "moniker": {}
                   }
-               },
-               "rootUri": "$URI{.}"
-            },
-            "jsonrpc": "2.0",
-            "id": 1,
-            "method": "initialize"
+               }
+            }
          },
          "wait": [
             {
-               "id": 1,
+               "id": "ada-1",
                "result": {
                   "capabilities": {
-                     "typeDefinitionProvider": true,
-                     "alsReferenceKinds": [
-                        "reference",
-                         "access",
-                        "write",
-                        "call",
-                        "dispatching call", "parent", "child"
-                     ],
-                     "hoverProvider": true,
-                     "definitionProvider": true,
-                     "renameProvider": {},
-                     "alsCalledByProvider": true,
-                     "referencesProvider": true,
                      "textDocumentSync": 2,
                      "completionProvider": {
                         "triggerCharacters": [
-                            ".",
-                            "("
+                           ".",
+                           "("
                         ],
                         "resolveProvider": false
                      },
-                     "documentSymbolProvider": {}
+                     "hoverProvider": true,
+                     "declarationProvider": true,
+                     "definitionProvider": true,
+                     "typeDefinitionProvider": true,
+                     "implementationProvider": true,
+                     "referencesProvider": true,
+                     "documentHighlightProvider": true,
+                     "documentSymbolProvider": {},
+                     "codeActionProvider": {},
+                     "documentFormattingProvider": true,
+                     "renameProvider": {},
+                     "foldingRangeProvider": true,
+                     "executeCommandProvider": {
+                        "commands": [
```

Comment:
I would remove this for two reasons: 1) they're not relevant for what we're testing and 2) every time we add a new command, we'll need to change these tests even if they don't have anything to do with the new command.

Reply:
Ok I'll remove it then